src/Blockcore.Indexer.Core/Controllers/QueryController.cs

```diff
@@ -39,6 +40,18 @@ public IActionResult GetAddress([MinLength(4)][MaxLength(100)] string address)
          return Ok(storage.AddressBalance(address));
       }
 
+      /// <summary>
+      /// Get the balance of multiple addresses.
+      /// </summary>
+      /// <param name="addresses"></param>
+      /// <returns></returns>
+      [HttpPost]
+      [Route("addressbalances")]
```

Comment:
This should follow the convention 
`address/balances` maybe? or 
"address/addressbalances"